c_src/scenic/ops/script_ops.h

```diff
@@ -0,0 +1,187 @@
+#pragma once
+#include "scenic_types.h"
+
+typedef enum {
+  script_op_draw_line = 0x01,
+  script_op_draw_triangle = 0x02,
+  script_op_draw_quad = 0x03,
+  script_op_draw_rect = 0x04,
+  script_op_draw_rrect = 0x05,
+  script_op_draw_arc = 0x06,
+  script_op_draw_sector = 0x07,
+  script_op_draw_circle = 0x08,
+  script_op_draw_ellipse = 0x09,
+  script_op_draw_text = 0x0A,
+  script_op_draw_sprites = 0x0B,
+  script_op_draw_script = 0x0F,
+
+  script_op_begin_path = 0x20,
+  script_op_close_path = 0x21,
+  script_op_fill_path = 0x22,
+  script_op_stroke_path = 0x23,
+  script_op_move_to = 0x26,
+  script_op_line_to = 0x27,
+  script_op_arc_to = 0x28,
+  script_op_bezier_to = 0x29,
+  script_op_quadratic_to = 0x2A,
+  //script_op_triangle = 0x2B,
+  //script_op_quad = 0x2C,
+  //script_op_rect = 0x2D,
+  //script_op_rrect = 0x2E,
+  //script_op_sector = 0x2F,
+  //script_op_circle = 0x30,
+  //script_op_ellipse = 0x31,
```

Comment:
We should probably either zorch these or add an explanation for why they're still here.

Reply:
Removed